docs/source/conf.py

```diff
@@ -288,16 +288,34 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
+
 html_theme_options = {
-    "show_toc_level": 2,
```

Comment:
This could be kept?

Reply:
I tested it without and found no difference. But might be missing something.

Reply:
I think it should influence the right-sidebar page TOC, by default it only shows the first level, and expands when scrolling down the page. With setting this, it should already show the first two levels by default (and then still further expands to more levels if those are present when scrolling)

Reply:
OK, I see it now ([this section for example](http://crossbow.voltrondata.com/pr_docs/36591/developers/guide/index.html)).
Will put it back in the next commit (accidentally removed it twice).

Reply:
[8b40a3e](https://github.com/apache/arrow/pull/36591/commits/8b40a3e10c0baed9516c5a2ce1d9aed4c030970e)